# This is a hunk fix for the 5.17 series.
# Original commit:  41ceaf5611bf5b9384e3f2aec5b591d5734126f9
# URL:  https://github.com/google/bbr/commit/41ceaf5611bf5b9384e3f2aec5b591d5734126f9

License header:

net/ipv4/tcp_rate.c:
// SPDX-License-Identifier: GPL-2.0-only

--- a/net/ipv4/tcp_rate.c.orig	2022-06-06 18:53:48.958249988 -0700
+++ b/net/ipv4/tcp_rate.c	2022-06-06 18:55:47.604253898 -0700
@@ -105,6 +105,7 @@ void tcp_rate_skb_delivered(struct sock
 	if (!rs->prior_delivered ||
 	    tcp_skb_sent_after(tx_tstamp, tp->first_tx_mstamp,
 			       scb->end_seq, rs->last_end_seq)) {
+		rs->prior_lost	     = scb->tx.lost;
 		rs->prior_delivered_ce  = scb->tx.delivered_ce;
 		rs->prior_delivered  = scb->tx.delivered;
 		rs->prior_mstamp     = scb->tx.delivered_mstamp;
